server/src/domain/repositories/asset.repository.ts

```diff
@@ -14,6 +14,7 @@ export interface AssetSearchOptions {
   trashedBefore?: Date;
   type?: AssetType;
   order?: 'ASC' | 'DESC';
+  withDeleted?: boolean;
```

Comment:
Is this also a convenient bug fix?

Reply:
Previously deleted assets were only included in the results when it was used in a specific request for trashed assets. This new option let's me additionally request them for the report query as well